src/SFGUI/Scrollbar.cpp

```diff
@@ -92,6 +92,26 @@ sf::Vector2f Scrollbar::CalculateRequisition() {
 	return sf::Vector2f( mimimum_slider_length, mimimum_slider_length );
 }
 
+void Scrollbar::HandleMouseWheelEvent(int delta, int x, int y) {
+	if (GetOrientation() == Orientation::VERTICAL) {
+		auto slider_rect = GetSliderRect();
```

Comment:
We don't like to declare locals if they are only used once. Move the calls into the line where they are used.